tree_planter.rb

```diff
@@ -199,7 +200,7 @@ def delete_branch(endpoint, repo_path, config_obj)
   end
   # rubocop:enable Metrics/AbcSize
 
-  # rubocop:disable Metrics/AbcSize, Metrics/ParameterLists, Metrics/PerceivedComplexity
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/ParameterLists, Metrics/PerceivedComplexity
```

Comment:
Finally fixed all of the rubocop issues. Are you okay with ignoring Metrics/CyclomaticComplexity on the deploy_tree method?

I don't really see a huge benefit to extracting a method(s) in there.

Reply:
Yep